pybossa/api/task_run.py

```diff
@@ -132,7 +132,21 @@ def _add_created_timestamp(self, taskrun, task, guard):
         guard._remove_task_stamped(task, get_user_id_or_ip())
 
     def _after_save(self, instance):
-        mark_if_complete(instance.task_id, instance.project_id)
+        project_id = instance.project_id
+        mark_if_complete(instance.task_id, project_id)
+
+        project = project_repo.get(project_id)
+        if not project.get_quiz_enabled(): return
```

Comment:
can the return be on a new line?

Reply:
we can also include not user.get_quiz_in_progress(project_id) here and return